src/content/admin/cloud-credentials/aws.mdx

```diff
@@ -69,8 +69,36 @@ export PROVIDER_ARN=arn:aws:iam::112233445566:oidc-provider/container.googleapis
 
 In this step, you'll create an IAM Role that allows access to S3 and configure it to assume the role using the OIDC identity provider you set up in the previous step.
 
-First, create the trust policy that allows the role to be assumed by the OIDC identity provider.
-Save the following content to a file called `trust-policy.json` (replace your environment variables):
+First, create the trust policy that allows the role to be assumed by the OIDC identity provider. To do this, create a local file named `trust-policy.json`,  copy the contents of the file below, and substitute the placeholder values with the resources that we created in the previous steps:
+
+
+```yaml title="trust-policy.json"
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "<PROVIDER_ARN>"
+      },
+      "Action": "sts:AssumeRoleWithWebIdentity",
+      "Condition": {
+        "StringEquals": {
+          "<OIDC_ISSUER>:aud": "<AUDIENCE>",
+          "<OIDC_ISSUER>:sub": "<OIDC_SUBJECT>"
+        }
+      }
+    }
+  ]
+}
+```
+
+- **OIDC_ISSUER**: The OIDC Issuer of your Okteto instance `OIDC_SUBJECT` that we retrieved in Step 1, without the "https://" part of the url. 
```

Comment:
@codyjlandstrom if you have a better idea of how to explain this, please change it. This is technically correct but it's not pretty.

Reply:
I made a couple small tweaks around this, but otherwise sounds good.